css/style.css

```
@@ -0,0 +1,721 @@
+  max-height: 600px;
+}
+
+.partner-organizer {
+  display: block;
+  position: relative;
+  width: 100%;
+  background-color: #272a32;
+}
+
+.grid-container {
+  display: grid;
+  grid-column-gap: 5px;
+  grid-template-columns: auto auto auto auto auto;
+  padding: 10px;
+  justify-content: center;
+  height: 334px;
+  align-items: center;
+}
+
+.grid-item {
+  margin: 0 0 0 3px;
+  padding: 0;
+  border: 0;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 230px;
+  height: 228px;
+  background: rgba(255, 255, 255, 0.1);
+  color: #fff;
+}
+
+.grid-item-img {
+  justify-content: center;
+  align-items: center;
+}
+
+.highligts {
+  color: #ec5242;
+  font-weight: 300;
+}
+
+.h2-link {
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  font-weight: 300;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.feature-title {
+  display: flex;
+  justify-content: center;
+  font-weight: 400;
+}
+
+/* JAVASCRIPT BEGIN */
+
+.speaker-conter {
+  display: flex;
+  justify-content: center;
+}
+
+.guest-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  height: 40%;
+  margin-top: 20px;
+}
+
+.guest-container img {
+  flex-basis: 30%;
+  max-width: 120px;
+  height: 120px;
+}
+
+.guest-container div {
+  flex-basis: 70%;
+  margin-left: 20px;
+}
+
+#guests li div {
+  margin-top: 0;
+  line-height: 25px;
+}
+
+#guests h3 {
+  font-family: 'Lato', sans-serif;
+  font-size: 12px;
+  color: #ec5242;
+}
+
+.img-radius {
+  border: 2px solid grey;
+  border-radius: 20px;
+}
+
+.content-spacing.bg-c-3 h2 {
+  margin-top: 8px;
+}
+
+.content-spacing.bg-c-3 p {
+  line-height: 23px;
+}
+
+.content-spacing.bg-c-3 img {
+  width: 100%;
+  margin-top: 25px;
+  border: 2px solid lightgray;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-container {
+  display: grid;
+  grid-column-gap: 10px;
+  grid-template-columns: auto auto;
+  flex-direction: row;
+  align-items: center;
+  gap: 12px;
+  margin: 35px 10px;
+}
+
+/* JAVASCRIPT FINISH */
+
+.partiner-tile {
+  display: flex;
+  justify-content: center;
+  color: #fff;
+  padding: 10px;
+  margin: 30px 0 0 0;
+}
+
+.grid-container-partner {
+  display: grid;
+  grid-column-gap: 55px;
+  grid-template-columns: auto auto auto auto auto;
+  padding: 10px;
+  justify-content: center;
+  align-items: center;
+  height: 50%;
+  background-color: #272a32;
+}
+
+.grid-item-partner {
+  margin: 0 0 0 3px;
+  padding: 0;
+  border: 0;
+  align-items: center;
+  justify-content: center;
+  width: 230px;
+  height: 228px;
+}
+
+.footer-organizer {
+  display: flex;
+  justify-content: space-evenly;
+  flex-direction: row;
+  align-items: center;
+}
+
+.footer-title {
+  display: flex;
+  margin-right: 450px;
+}
+
+.footer-paragraph {
+  font-weight: 300;
+}
+
+@media only screen
+  and (min-device-width: 200px)
+  and (max-device-width: 768px)
+  and (-webkit-min-device-pixel-ratio: 2) {
```

[Optional]
Making media queries with device width is not a suggested way. If you're making a responsive website, you should use min-width/max-width in your media queries rather than min-device-width/max-device-width in order to target a wider range of screen sizes.
 ( [The device-width media feature is deprecated. It should be avoided](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/device-width).)

```suggestion
@media only screen  and (max-width: 768px)
  {
```